includes/class-gistpress.php

```diff
@@ -420,6 +420,9 @@ public function get_gist_html( $url, array $args ) {
 		$this->debug_log( '<strong>' . __( 'Raw Key (Transient & Post Meta):', 'gistpress' ) . '</strong> ' . $raw_key, $shortcode_hash );
 		$this->debug_log( '<strong>' . __( 'Processed Output Key (Transient):', 'gistpress' ) . '</strong> ' . $transient_key, $shortcode_hash );
 
+		// Strip unneccessary elements from HTML output
+		$html = preg_replace('/^<!DOCTYPE.+?>/', '', str_replace( array('<html>', '</html>', '<body>', '</body>'), '', $html));
```

Comment:
I've not found that situation in my own testing where PHP produced a lower case doctype, but better safe than sorry. I've updated the pull request to make it case insensitive just in case. For the same reason, I've also replaced `str_replace` with `str_ireplace` to make the text replacement case insensitive as well. Thanks for pointing it out